qsr_lib/scripts/example_ros_client.py

```diff
@@ -34,7 +34,8 @@
                "qtcbc": "qtc_bc_simplified",
                "rcc3a": "rcc3_rectangle_bounding_boxes_2d",
                "arg_distance": "arg_relations_distance",
-                "mos": "moving_or_stationary"}
+                "mos": "moving_or_stationary",
+                "multiple": ("cone_direction_bounding_boxes_centroid_2d", "rcc3_rectangle_bounding_boxes_2d", "moving_or_stationary", "qtc_b_simplified")}
```

Comment:
If you delete multiple from options then you can add it as follows in a new line:
```python
options["multiple"] = options.values()